LidarrAPI/LidarrAPI.csproj

```diff
@@ -1,23 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <Folder Include="wwwroot\"/>
+    <Folder Include="wwwroot\" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="DogStatsD-CSharp-Client" Version="3.1.0"/>
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.5"/>
-    <PackageReference Include="NLog" Version="5.0.0-beta11"/>
-    <PackageReference Include="NLog.Web.AspNetCore" Version="4.4.1"/>
-    <PackageReference Include="Octokit" Version="0.29.0"/>
-    <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="2.0.1"/>
-    <PackageReference Include="TraktApiSharp" Version="1.0.0-alpha2"/>
-    <PackageReference Include="SharpRaven" Version="2.3.2"/>
-  </ItemGroup>
-  <ItemGroup>
-    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.0"/>
-    <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0"/>
+    <PackageReference Include="DogStatsD-CSharp-Client" Version="3.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Used to be AspNetCore.All? Name change on package? Also do we need to explicit call a version here?

Reply:
Yeah the package name changed and I think the dependencies it brings in did as well. The docs I saw recommended no version, but I don't know a ton about packages in .NET. You can see the second and third bullets [here](https://docs.microsoft.com/en-us/aspnet/core/migration/20_21?view=aspnetcore-2.1).